lib/p5.speech.js

```diff
@@ -358,14 +355,33 @@
   // this one 'start' cycle.  if you need to recognize speech more
   // than once, use continuous mode rather than firing start()
   // multiple times in a single script.
-  p5.SpeechRec.prototype.start = function() {
+  p5.SpeechRec.prototype.start = function(_continuous, _interimResults) {
     if('webkitSpeechRecognition' in window) {
-      this.rec.continuous = this.continuous;
-      this.rec.interimResults = this.interimResults;
+      this.rec.continuous = this.continuous = _continuous === undefined ? this.continuous : _continuous;
```

Comment:
As discussed in #10 with @rev3rend the `continuous` property should be set to false by default. This was my mistake in the Coding Train discussion yesterday.